src/com/zombiecastlerush/role/Enemy.java

```diff
@@ -15,7 +17,9 @@ public Enemy(int id, Room room){
     public void run() {
         //TODO: add zombie/robot behaviors
         try {
-            while(true) {
+            long timerStart = System.currentTimeMillis();
```

Comment:
I would actually propose to use a keystroke/text input end (EG, press Enter to end), because I think this would be closer to what the Game class will do to end the game. I think this would be better than using a timer.

Reply:
Actually, I realize I read this wrong. Leaving the comment here for posterity, but per comment above, are we sure we want to implement Runnables for this game?